src/components/veocean/LockOcean.svelte

```diff
@@ -190,6 +190,17 @@
   };
 
   $: calculatedMultiplier, $form.unlockDate, updateMultiplier();
+
+  const updateFormAmount = () => {
+    let _amount = $form.amount;
+    _amount = _amount == null ? 0 : parseInt(_amount);
+    _amount = _amount < 0 ? 0 : parseInt(_amount);
+    
+    // Do not enforce max balance, let user play
+    // _amount = _amount > oceanBalance ? oceanBalance : parseInt(_amount);
```

Comment:
I agree with not enforcing the amount to be less than the max amount inside input as long as we have an error message for that case. Right now there isn't any message to signal the user that he needs to change the amount in order to be allowed to continue the lock process.
Unless we have a clear message for when user exceeds the max amount I suggest we enable the enforcing.

Reply:
Ok, then let's enforce. I would rather keep it minimal and avoid error messages. You can only select correct values.